### ISSUE TITLE
Guidance on working with reader macros

### ISSUE THREAD

**Issue by martinklepsch** [borkdude/rewrite-edn#37]
Is it in scope for this lib to support transforms involving reader macros? I'm not sure how this would work so totally understand if it's out of scope :) 
```clj
(def z (r/parse-string "{:data {:foo #profile {:prod 1 :dev 2}}}"))
(r/get-in z [:data :foo]) ;-> <reader-macro: #profile {:prod 1 :dev 2}>
(r/assoc-in z [:data :foo :prod] false) ;-> NPE
```

**Comment by borkdude** [comment]
Show me how you would use `get-in` with normal EDN that contains a reader macro

**Comment by martinklepsch** [comment]
> Show me how you would use `get-in` with normal EDN that contains a reader macro

Yeah I don't really know how that would work either 😅  I was just thinking of transforming [aero](https://github.com/juxt/aero) configs using rewrite-edn so that's where that line of thinking is coming from...

**Comment by borkdude** [comment]
I don't know the answer either but would be happy to hear what you came up with :)

**Comment by martinklepsch** [comment]
Actually, thinking about this more, couldn't there just be a special way of "entering" the value behind the reader macro? I guess that's possible with rewrite-clj already... 💡  I think I just need dive into `rewrite-clj` a bit more :)

Feel free to close this :)

**Comment by borkdude** [comment]
Yes, I guess we could support that, e.g. `(get-in [:foo :bar ?? :prod])` but what would go on the question marks?

**Comment by martinklepsch** [comment]
That's the question, isn't it? 😀  Maybe something like this?

```clj
(r/get-in [:foo :bar (r/reader-tag 'profile) :prod])
```

**Comment by borkdude** [comment]
I'd be willing to support that
